### PR TITLE
Resolved #2001, BULK Housing grant exemption finder

### LINKED ISSUE #2001: MFIP housing grant finder
BULK list that will assist in identifying cases that are eligible for housing grant because they meet an exemption. Script will pull cases into Excel that meet an exemption/potential exemption, and that are coded as "Y" for sub housing on the SHEL panel. The script will also pull in the payments issued over the rolling 12 months so that supplements/overpayments can be identified with ease.

### PR DESCRIPTION
**Pull request by IlseFerris**
New script that creates a list the rolling 12 months of housing grant issuances for MFIP recipients who've met an exemption. 

I have instructions made. Will add them to SIR before the end of the week.

### CODE REVIEW

File: Script Files/BULK/BULK - HOUSING GRANT EXEMPTION FINDER.vbs [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+		Set run_another_script_fso = CreateObject("Scripting.FileSystemObject")
+		Set fso_command = run_another_script_fso.OpenTextFile(FuncLib_URL)
+		text_from_the_other_script = fso_command.ReadAll
+		fso_command.Close
+		Execute text_from_the_other_script
+	END IF
+END IF
+'END FUNCTIONS LIBRARY BLOCK================================================================================================
+
+'Required for statistical purposes==========================================================================================
```
Comment (theVKC) [inline Script Files/BULK/BULK - HOUSING GRANT EXEMPTION FINDER.vbs]: We moved this, @IlseFerris...

**Comment by theVKC** [comment]
Looks great, except the stats block isn't the new combined one with the top stats info. Can you combine these? Thanks!

**Comment by IlseFerris** [comment]
I updated the stats information. Tested again, and ready to go!

**Comment by theVKC** [comment]
I saw that. I set up an alert for your commits. :P

**Comment by IlseFerris** [comment]
Are you watching me? LOL! But seriously....

**Pull request merged**